Packs/Malwarebytes/ReleaseNotes/1_1_9.md

```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Malwarebytes
+- Fixed an issue where ***!malwarebytes-list-endpoint-info*** command would fail when finding endpoint's model.
```

Comment:
```suggestion
- Fixed an issue where ***!malwarebytes-list-endpoint-info*** command would fail when finding endpoint's model.
- Updated Docker image to demisto/oauthlib:1.0.0.36344
```